proto/cometbft/abci/v2/types.proto

```diff
@@ -17,6 +17,8 @@ import "google/protobuf/timestamp.proto";
 //----------------------------------------
 // Request types
 
+// Handling v1 as v2: REJECT
```

Comment:
Just to clarify, does this mean that, if we receive `v1.Request`, but we're expecting `v2.Request`, we reject it?

Reply:
Exactly. And "reject" means:

* either "it doesn't make sense" (this is the case here)
* or it is an error condition

Let me know if you think it is worth distinguishing between REJECT and ERROR